CHANGELOG.md

```diff
@@ -109,7 +110,7 @@ variable "rds_security_group_id" {
   rds_security_group_id    = var.rds_security_group_id
   ## OPTIONAL
   db_admin_username                                    = "postgres"
-  default_multipart_chunksize_mb                       = 250
+  s3MultipartChunksizeMb                               = 250
```

Comment:
Need to revert this back to `default_multipart_chunksize_mb` in this section since we are not changing any of the main terraform values.

Reply:
Reverted